src/main/resources/io/jenkins/plugins/extratoolinstallers/installers/AnyOfInstaller/config.jelly

```diff
@@ -1,7 +1,16 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler" xmlns:t="/hudson/tools">
-    <f:property field="installers"/>
-    <f:advanced>
+    <tr>
+        <td class="setting-leftspace"><st:nbsp/></td>
+        <td class="setting-main" colspan="2">
+            <table>
+                <f:property field="installers"/>
+            </table>
+        </td>
+        <f:helpLink url="/descriptor/io.jenkins.plugins.extratoolinstallers.installers.AnyOfInstaller/help/installers" featureName="installers"/>
+    </tr>
+    <f:helpArea />
```

Comment:
@oleg-nenashev This is the only code change I'm a little uncertain about.  I'm not an expert with Jelly (or HTML, or CSS) so this is not something I'm confident with.

My intention here is that I want the "installers" field (which is a List of ToolInstallers) to have its own help icon (backed by help text), but for the installers themselves to spread over both the entry "title" column and the data fields as well. e.g.
```
 ----------- ------- ---
| Installer........ |   |
| .........List.... | ? |
| .............Here |   |
| Retries           |   |
 ----------- ------- ---
```
which, when "Retries" is clicked on, expands to be
```
 ----------- ------- ---
| Installer........ |   |
| .........List.... | ? |
| .............Here |   |
| fieldName | value | ? |
| fieldName | value | ? |
 ----------- ------- ---
```
I did a lot of trial-and-error before I got to this point, but this is doing a copy/paste of what `entry.jelly` (from the Jenkins core) does, and I'm never entirely happy with copy/paste code.
i.e. At present, this works exactly as expected, but if entry.jelly ever changes significantly, this will stop working as expected and may start looking really ugly instead.

I did try wrapping it in a `<f:entry>` box but this meant that, when the `<f:advanced>` block was opened, the installers got squashed up like this:
```
 ----------- ------- ---
|             Insta |   |
|             List  | ? |
|             Here  |   |
| fieldName | value | ? |
| fieldName | value | ? |
 ----------- ------- ---
```


If you know of any better ways of doing this, please let me know.

Reply:
@oleg-nenashev If you know of any other Jenkins/Jelly experts who might have an opinion on this, do let me know.

Reply:
Pinging @jenkinsci/code-reviewers is a way to go in such case.
W.r.t this change, I would not use help area. This is a hack, and this control is supposed to be used only inside Jenlly controls. 

The best way would be to actually stop using table rendering in this Jelly file, because Tables usually produce a lot of rendering issues